httpx/concurrency/asyncio/backend.py

```diff
@@ -112,10 +107,10 @@ def write_no_block(self, data: bytes) -> None:
                     raise WriteTimeout() from None
 
     def is_connection_dropped(self) -> bool:
-        return self.stream_reader.at_eof()
+        return self.stream.at_eof()
 
     async def close(self) -> None:
-        self.stream_writer.close()
+        await self.stream.close()
```

Comment:
This `await` should be the only real functional change (at least for Python < 3.8). I get spurious errors in tests from this, though, that look a lot like [this](https://github.com/aio-libs/aiohttp/issues/3535) bug that is supposed to be fixed in Python 3.7.4 which I'm running 🤔